### PR TITLE
Feature/internationalization

### PR DESCRIPTION
**Pull request by pedromtec**
#### What problem is this solving?

I need to add i18n and send the locale to the iframe.  

#### How should this be manually tested?

[Workspace](https://teste--checkoutio.myvtex.com/payment)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: react/Payment.tsx [hunk 1]
```diff
@@ -24,19 +26,24 @@ if (LOCAL_IFRAME_DEVELOPMENT) {
 }
 
 const Payment: React.FC = () => {
+  const [iframeLoading, setIframeLoading] = useState(true)
   const [cardData, setCardData] = useState<Card | null>(null)
   const iframeRef = useRef<HTMLIFrameElement>(null)
 
+  const runtime = useRuntime()
+  const { locale } = runtime.culture
```
Comment (marcoskwkm) [inline react/Payment.tsx]:
```suggestion
  const { culture: { locale } } = runtime
```

File: react/styles.css [hunk 2]
```diff
@@ -1 +1,3 @@
-.container {}
+:global(.vtex__icon-spinner) {
+    position: aboslute;
```
Comment (marcoskwkm) [inline react/styles.css]: This can't be working 😆 
```suggestion
    position: absolute;
```

File: react/typings/vtex.styleguide.d.ts [hunk 3]
```diff
@@ -1,9 +1 @@
-declare module 'vtex.styleguide' {
-  import { ComponentType } from 'react'
-
-  export const Input: ComponentType<InputProps>
-
-  interface InputProps {
-    [key: string]: any
-  }
-}
+declare module 'vtex.styleguide'
```
Comment (marcoskwkm) [inline react/typings/vtex.styleguide.d.ts]: Do you need this?

File: react/styles.css (hunk 2 above)
Comment (pedromtec) [reply react/styles.css]: I will improve the css. It's not last solution for this project.

File: react/typings/vtex.styleguide.d.ts (hunk 3 above)
Comment (pedromtec) [reply react/typings/vtex.styleguide.d.ts]: Yes, several projects are using types for the styleguide. Look: https://github.com/vtex-apps/product-list/blob/master/react/typings/vtex.styleguide.d.ts . The project doesn’t work without it

**Comment by marcoskwkm** [comment]
Anything to say about the other comments I left?

File: react/Payment.tsx [hunk 4]
```diff
@@ -54,16 +63,31 @@ const Payment: React.FC = () => {
...
-      />
-      {cardData && <p>{JSON.stringify(cardData)}</p>}
+    <div className="relative">
+      {iframeLoading && (
+        <div className="absolute top-0 left-0 right-0 bottom-0 bg-white-70 z-1 flex items-center justify-center">
+          <Spinner />
+        </div>
+      )}
+      <div className="flex-none">
+        <div className="">
```
Comment (lucasecdb) [inline react/Payment.tsx]: is this className needed?
```suggestion
        <div>
```

File: react/typings/vtex.styleguide.d.ts (hunk 3 above)
Comment (marcoskwkm) [reply react/typings/vtex.styleguide.d.ts]: This actually does not add any types to your project at all, that's why I asked. But I noticed this makes `tsc` stop complaining, so I'll let this pass.

File: react/styles.css (hunk 2 above)
Comment (pedromtec) [reply react/styles.css]: Where can I add a comment ?

Comment (pedromtec) [reply react/styles.css]: I wanna render my app only when the Stylesheets was created inside of card-form-ui. 
I'm currently using `<Spinner />` but I don't know if it's the best way to solve this UI problem. But maybe this change should not be part of this PR.

Comment (marcoskwkm) [reply react/styles.css]: You confused me with your comments lol. This is fine now that you fixed it 😆

File: react/Payment.tsx [hunk 5]
```diff
@@ -54,16 +63,31 @@ const Payment: React.FC = () => {
...
-        ref={iframeRef}
-      />
-      {cardData && <p>{JSON.stringify(cardData)}</p>}
+    <div className="relative">
+      {iframeLoading && (
+        <div className="absolute top-0 left-0 right-0 bottom-0 bg-white-70 z-1 flex items-center justify-center">
+          <Spinner />
+        </div>
+      )}
+      <div className="flex-none">
```
Comment (lucasecdb) [inline react/Payment.tsx]: do you need this class? the parent div isn't flexbox.

File: react/Payment.tsx [hunk 6]
```diff
@@ -54,16 +63,31 @@ const Payment: React.FC = () => {
...
+        <div>
```
Comment (lucasecdb) [inline react/Payment.tsx]: I think this div is also unnecessary

File: react/typings/vtex.styleguide.d.ts (hunk 3 above)
Comment (lucasecdb) [inline react/typings/vtex.styleguide.d.ts]: can you use [module augmentation](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) for this? so we can benefit of all the types that exist in the styleguide package

File: react/typings/vtex.styleguide.d.ts [hunk 7]
```diff
@@ -1 +1,7 @@
-declare module 'vtex.styleguide'
+declare module 'vtex.styleguide' {
```
Comment (lucasecdb) [inline react/typings/vtex.styleguide.d.ts]: add this
```ts
import * as Styleguide from 'vtex.styleguide'
```
so we can benefit from module augmentation

**Pull request merged**